recipe/meta.yaml

```diff
@@ -24,7 +24,7 @@ source:
 
 build:
   number: {{ build_number }}
-  skip: true  # [win and vc<14]
+  skip: true  # [(win and vc<14) or variant=="hcc"]
```

Comment:
Why?  Does the hcc variant only make sense if the rocm patches are applied?

Reply:
> Does the hcc variant only make sense if the rocm patches are applied?

Yes.